ui/tests/integration/components/clients/usage-stats-test.js

```diff
@@ -11,80 +11,20 @@ import { hbs } from 'ember-cli-htmlbars';
 module('Integration | Component | clients/usage-stats', function (hooks) {
   setupRenderingTest(hooks);
 
-  hooks.beforeEach(function () {
-    this.showSecretSyncs = false;
-    this.counts = {};
-
-    this.renderComponent = async () =>
-      await render(
-        hbs`<Clients::UsageStats @title="My stats" @totalUsageCounts={{this.counts}} @showSecretSyncs={{this.showSecretSyncs}} />`
-      );
-  });
-
-  test('it renders defaults', async function (assert) {
-    await this.renderComponent();
-
-    assert.dom('[data-test-stat-text]').exists({ count: 4 }, 'Renders 4 Stat texts even with no data passed');
-    assert.dom('[data-test-stat-text="Total clients"]').exists('Total clients exists');
-    assert.dom('[data-test-stat-text="Total clients"] .stat-value').hasText('-', 'renders dash when no data');
-    assert.dom('[data-test-stat-text="Entity"]').exists('Entity exists');
-    assert.dom('[data-test-stat-text="Entity"] .stat-value').hasText('-', 'renders dash when no data');
-    assert.dom('[data-test-stat-text="Non-entity"]').exists('Non entity clients exists');
-    assert.dom('[data-test-stat-text="Non-entity"] .stat-value').hasText('-', 'renders dash when no data');
+  test('it renders', async function (assert) {
+    await render(
+      hbs`
+        <Clients::UsageStats @title="My stats">
+         yielded content!
+        </Clients::UsageStats>`
+    );
+
+    assert.dom('[data-test-usage-stats="My stats"]').exists();
```

Comment:
do we care about testing that the `@title` `@description` or docs link are rendered as well? i know most of the value of this component comes from the yielded content, but might be nice to have a test that asserts that the basics are there. 🙂

Reply:
yeah! i can definitely add that 😄